AirCasting/Utils/Averaging.swift

```diff
@@ -0,0 +1,151 @@
+// Created by Lunar on 29/09/2021.
+//
+
+import Foundation
+import Combine
+import CoreData
+import Algorithms
+
+struct AvgMeasurement {
+    let window: AveragingWindow = .zeroWindow
+    let treshold: TimeThreshold
+}
+
+enum AveragingWindow: Int {
+    case zeroWindow = 1
+    case firstThresholdWindow = 5
+    case secondThresholdWindow = 60
+}
+
+enum TimeThreshold: Int {
+    // Two hours: 60 * 60 * 2 = 7200
+    case firstThreshold = 7200
+    // Nine hours: 60 * 60 * 9 = 32400
+    case secondThreshold = 32400
+}
+
+final class Averaging: NSObject, ObservableObject {
+    // TODO:
+    // 1. calculate average
+    // 2. remove unaveraged measurements
+    // 3. mark measurements as averaged
+    // 4. setup timer for new session
+    // 5. perform averaging of previous measurements after crossing threshold
+    // 5. get proper averaging window (for threshold)
```

Comment:
I think all but last are done? Now we only have to make it "universal" for both thresholds

Reply:
I removed unnecessary comments and copied-pasted uoour description oof averaging from android code :)